Sources/MessagingPush/UserNotificationsFramework/Wrappers.swift

```diff
@@ -120,6 +120,10 @@ public struct UNNotificationWrapper: PushNotification {
 class UNUserNotificationCenterDelegateWrapper: PushEventHandler {
     private let delegate: UNUserNotificationCenterDelegate
 
+    var identifier: String {
+        String(describing: delegate)
```

Comment:
By returning the instance description of the nested delegate, this is what will uniquely identify `UNUserNotificationCenterDelegateWrapper` instances from 1 another. Since `UNUserNotificationCenterDelegateWrapper` is just a wrapper, it's the nested delegate that we really care about.